### DISCUSSION TITLE
Link Already Created Account and Send File

### DISCUSSION THREAD

**Discussion by rohitarya** [AsamK/signal-cli#467] [Q&A]
Hi,
I want to link my already created account to the cli how can I do this?
Also how we send file to recepient from the cli?

**Comment by jasonho1308** [comment]
`signal-cli -u USERNAME register`
To register your created account to the computer (USERNAME is your phone number, with country code)
`signal-cli -u USERNAME verify CODE`
To verify
`signal-cli -u USERNAME send -m "This is a message" [RECIPIENT [RECIPIENT …​]] [-a [ATTACHMENT [ATTACHMENT …​]]]`
To send message
